content/en/containers/kubernetes/configuration.md

```diff
@@ -592,6 +592,70 @@ For a complete list of environment variables for the Helm chart, see the [full l
 {{% /tab %}}
 {{< /tabs >}}
 
+## Environment variables
+The containerized Datadog Agent can be configured by using environment variables, for an extensive list of supported environement variables in the [Environment Variables][26] section of the Docker Agent documentation.
+
+### Examples
+{{< tabs >}}
+
+{{% tab "Datadog Operator" %}}
+When using the Datadog Operator, environment variables are set in the override in the context of a component or container as `[key].containers.[key].env []object` or `[key].env []object` with the following possible keys: `nodeAgent`, `clusterAgent` or `clusterChecksRunner` The container level settings take priority over any component level settings.
```

Comment:
```suggestion
When using the Datadog Operator, you can set additional environment variables in `override` for a component with `[key].env []object`, or for a container with `[key].containers.[key].env []object`. The following keys are supported: 

- `nodeAgent`
- `clusterAgent`
- `clusterChecksRunner`

Container-level settings take priority over any component-level settings.
```

Reply:
Suggesting to reword and reformat this for clarity, but please confirm these edits are accurate.